crates/ruma-identifiers-validation/src/error.rs

```diff
@@ -77,7 +77,7 @@ pub enum MxcUriError {
     /// Media identifier malformed due to invalid characters detected.
     ///
     /// Valid characters are (in regex notation) `[A-Za-z0-9_-]+`.
-    /// See [here](https://spec.matrix.org/v1.4/client-server-api/#security-considerations-5) for more details.
+    /// See [here](https://spec.matrix.org/latest/client-server-api/#security-considerations-5) for more details.
```

Comment:
Same here, I see a very high chance this link will refer to the wrong place in the future.